app/components/AddTasks/ColorSelector.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
+
+import { useState } from 'react';
+// we'll need to get the current theme from the app store
+//import { useStore } from '@/lib/store/app';
+
+// constant to record colors that map to tailwind tokens
+export const COLORS = [
+  { name: 'red', class: 'bg-red-200' },
+  { name: 'blue', class: 'bg-blue-200' },
+  { name: 'green', class: 'bg-green-200' },
+  { name: 'yellow', class: 'bg-yellow-200' },
+  { name: 'purple', class: 'bg-purple-200' },
+  { name: 'pink', class: 'bg-pink-200' },
+  { name: 'orange', class: 'bg-orange-200' },
+  { name: 'gray', class: 'bg-gray-200' },
+];
+
+const ColorSelector = ({
+  onColorSelect,
+}: {
+  onColorSelect: (color: string) => void;
+}) => {
+  //TODO: create different sets of colors for tasks/notes depending on the theme
+  // const { theme } = useStore();
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+  const [selectedColor, setSelectedColor] = useState(COLORS[2].class);
+
+  return (
+    <div className="relative">
+      <button
+        className={`w-6 h-6 rounded-full border focus:outline-none ${selectedColor}`}
+        onClick={() => setDropdownOpen(!dropdownOpen)}
+      ></button>
```

Comment:
I don't love how this button is sitting 

<img width="468" alt="image" src="https://github.com/user-attachments/assets/be263b73-071b-4d08-8b31-fb5f71ad2689">

It should be more centered